cypress/e2e/modal-overlays.spec.js

```diff
@@ -0,0 +1,29 @@
+/// <reference types="cypress" />
+
+describe('Modal&Overlays', ()=>{
+    
+
+    it('Hover the element without clicking it', ()=> {
+        cy.visit('/pages/modal-overlays/popover').wait(3000)
+        cy.get('nb-card-body button').contains('Right').trigger("mouseenter").then(el=>{
```

Comment:
```suggestion
        cy.get('nb-card-body button').contains('Right').trigger("mouseenter")
```
Навіщо тут `.then` ? "el" далі ніде не використовується